discord/mentions.py

```diff
@@ -118,14 +118,14 @@ def to_dict(self) -> AllowedMentionsPayload:
         if self.everyone:
             parse.append("everyone")
 
-        if self.users == True:
+        if self.users:
             parse.append("users")
-        elif self.users != False:
+        elif not self.users:
```

Comment:
This seems to have been like this to allow for the passing of a list of users, so it should probably remain close to the same, e.g.
```py
if self.users is True:
    parse.append("users")
elif self.users is not False:
```
*Github won't allow suggestions on this because of the deleted line*